.github/workflows/ci.yml

```diff
@@ -252,6 +274,16 @@ jobs:
           cat /etc/hosts
         shell: bash
 
+      - name: Prepare env for Alpine Linux
+        if: ${{ contains(matrix.container, 'alpine') }}
+        run: |
+          apk add moreutils
+          # comment out "::1 localhost ..." to avoid conflicts with proxy tests
+          sed '/^::1/ s/^/#/' /etc/hosts | sponge /etc/hosts
+          python3 -m venv .venv --system-site-packages
```

Comment:
is passing `--system-site-packages` ok? installing `mitmproxy` with apk takes a few seconds, whereas building it from scratch for venv took ~4 minutes.

Reply:
It is fine.